File: 프로그래머스/2/42584. 주식가격/주식가격.py
from collections import deque
def solution(prices):
    
    arr = deque(prices)
    result = []
    while len(arr) > 0:
        n = arr.popleft()
        count = 0
        
        for i in arr:
            count += 1
            if n > i:
                break
                
        result.append(count)
        
    return result

File: 프로그래머스/2/42584. 주식가격/주식가격.java
import java.util.*;
class Solution {
    public List<Integer> solution(int[] prices) {
        List<Integer> answer = new ArrayList<Integer>();
        
        for(int i = 0; i < prices.length; i++){
            int count = 0;
            for(int j = i+1; j < prices.length; j++){
                count ++;
                if(prices[i] > prices[j]){
                    break;
                }
            }
            answer.add(count);
        }
        return answer;
    }
}